inventory/by_team/dls

```diff
@@ -0,0 +1,21 @@
+[dls:children]
+abid_production
+abid_staging
+dpul_production
+dpul_staging
+figgy_production
+figgy_staging
+# imagecat inventory should be added here
```

Comment:
imagecat is running on the nomad cluster

Reply:
Addressed in 801d805cbc5c84070c19b264d1ec5e58de06871d.